src/components/TextInput/TextInput.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Input, Label } from './styles';
+
+const TextInput = ({ label }) => {
```

Comment:
It's not `TextInput` because you can type only numbers here. It's `NumberField`. Also, why user couldn't provide a default value through props?

Reply:
Also, why there is no `onChange` callback (prop). So, how consumer of this component will take a value from input?

Reply:
> Also, why there is no `onChange` callback (prop). So, how consumer of this component will take a value from input?

I know that new code isn't good

Reply:
> Also, why there is no `onChange` callback (prop). So, how consumer of this component will take a value from input?

But I don't know how to do it better yet